src/internal/io.js

```diff
@@ -153,6 +155,16 @@ export function flush(channel) {
   return effect(FLUSH, channel)
 }
 
+export function getContext(prop) {
+  check(prop, is.string, `getContext(prop): argument ${ prop } is not a string`)
+  return effect(GET_CONTEXT, prop) // TODO: maybe accept functions also?
```

Comment:
Since we do `Object.assign` and not deep merge, I'd view `context` as a flat object. Thus not sure if complex accessors are needed. Maybe I'm missing some use cases, but I'd rather add such features on demand.